assets/app/view/game/spreadsheet.rb

```diff
@@ -360,7 +378,7 @@ def sorted_corporations
             when :market_shares
               num_shares_of(@game.share_pool, corporation)
             when :share_price
-              [corporation.share_price&.price || 0, -operating_order]
+              [corporation.share_price&.price || 0, -operating_order[1]]
```

Comment:
```suggestion
              [corporation.share_price&.price || 0, operating_order[0], -operating_order[1]]
```

Without `operating_order[0]`:

![Screenshot 2022-11-26 123049](https://user-images.githubusercontent.com/1045173/204106107-b007f629-3cbe-4063-86db-61b7e48f8871.png)

With it:

![Screenshot 2022-11-26 123137](https://user-images.githubusercontent.com/1045173/204106114-f57e1374-5f3f-49bb-8833-f93a0bb07ba1.png)

Reply:
thanks for catching :smile: